e2e/dir_scan_test.go

```diff
@@ -61,6 +61,8 @@ var _ = ginkgo.Describe("Running a SBOM and plugin scan", func() {
 				apitypes.Asset{
 					AssetInfo: &assetType,
 					FirstSeen: to.Ptr(time.Now()),
+					// Set to future time so asset is not terminated by discoverer.
+					LastSeen: to.Ptr(time.Now().Add(time.Minute * 10)),
```

Comment:
would setting the same time here work the same way as it is now (if both firstSeen and lastSeen are the same)?

Reply:
https://github.com/openclarity/vmclarity/blob/main/orchestrator/discoverer/discoverer.go#L165 if the `lastseen` time is lower than the discovery time, it will mark it as terminated